tests/devices/system_tests/test_undulator_system.py

```diff
@@ -1,16 +1,18 @@
 import pytest
+from ophyd_async.core import DeviceCollector
 
 from dodal.devices.undulator import Undulator
 
 SIM_INSERTION_PREFIX = "SR03S"
 
 
 @pytest.fixture
-def undulator():
-    undulator = Undulator(f"{SIM_INSERTION_PREFIX}-MO-SERVC-01:", name="undulator")
+async def undulator() -> Undulator:
+    with DeviceCollector():
+        undulator = Undulator(f"{SIM_INSERTION_PREFIX}-MO-SERVC-01:")
     return undulator
 
 
 @pytest.mark.s03
-def test_undulator_connects(undulator):
-    undulator.wait_for_connection()
+def test_undulator_connects(undulator: Undulator):
+    ...
```

Comment:
Should: Given this is now just rolled into the above and does nothing let's remove it

Reply:
You mean just remove the fixture and do the connect in this test?